processors/parseSchema.mjs

```diff
@@ -66,5 +66,6 @@ export default {
     randomed: false,
     repicked: false,
     pred_vict: false,
+    neutral_token: [],
```

Comment:
The column also needs to be added to the database schemas create_tables.sql/cql in order to be saved.

Reply:
Acfually since those files live in odota/core we'll have to make them separately. But without those changes the data will not appear on OpenDota, only for users running the parser independently.